### PR TITLE
Analytics/Report Fixes

### PR DESCRIPTION
**Pull request by jjennifergu**
## Overview

Updates after merging Sergio's analytics/reports PR from last year

## Changes Made

- Analytics: re-download GoogleService-Info.plist
- Reports: update endpoint, report model, error handling

## Test Coverage

- Verify analytics with Matt
- Send reports then verify through GET request in Postman

## Related PRs or Issues

Sergio's PR: https://github.com/cuappdev/eatery-blue-ios/pull/6

## Screenshots

https://user-images.githubusercontent.com/57200368/233798918-4ee0aae7-cb3f-4ecc-8f7a-29abc10bf029.mp4

### CODE REVIEW

**Review by antoinztte — APPROVED** [review]
LGTM

File: .DS_Store
Comment (NgaiJustin) [inline .DS_Store:1]: Add to `.gitignore` please.

File: Eatery Blue/.DS_Store
Comment (NgaiJustin) [inline Eatery Blue/.DS_Store:1]: As above

**Review by NgaiJustin — APPROVED** [review]
Yup lgtm.

**Review by ReadeP — APPROVED** [review]

**Pull request merged**